scripts/CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ elseif(UNIX)
         set(CMAKE_INSTALL_RPATH "@loader_path/../../../")
     else()
         set(ONEDAL_PY_LINK_OPTIONS "-Wl,-z,noexecstack,-z,relro,-z,now,-fstack-protector-strong,-fno-strict-overflow,-fno-delete-null-pointer-checks,-fwrapv")
-        set(CMAKE_INSTALL_RPATH "$ORIGIN/../../../")
+        set(CMAKE_INSTALL_RPATH "$ORIGIN/../../../:${oneDAL_LIBRARY_DIR}")

```

Comment:
Usage of absolute paths has two issues if package is built for public distribution:
- It is non-portable
- It is absolute path on building machine which can be a security issue

Reply:
Thanks for looking into it.

What would you think of enabling it for non-release builds through an optional argument?

I think it shouldn't be much of an issue that it isn't portable, because it would still be able to load the library if it finds it elsewhere (e.g. in the first RPATH entry which is relative, or in `LD_LIBRARY_PATH`).

Reply:
Optional argument is fine.

Reply:
More over we would need to ensure that we are not adding those accidentally to release builds

Reply:
Made it into an optional behavior that can be triggered by passing an argument `--dev` to the `setup.py` script.

I think this should be a sufficiently high barrier to avoid accidents, because it's not a recognized built-in argument for setuptools and thus calls to setup.py with that argument would error out before this PR.